src/Cake.Common/Tools/GitLink/GitLink3Aliases.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.GitLink
+{
+    /// <summary>
+    /// <para>Contains functionality related to <see href="https://github.com/gittools/gitlink">GitLink</see> version 3.</para>
+    /// <para>
+    /// In order to use the commands for this alias, include the following in your build.cake file to download and
+    /// install from NuGet.org, or specify the ToolPath within the <see cref="GitLink3Settings" /> class:
+    /// <code>
```

Comment:
@mholo65 sooooo I just copied what was in: https://github.com/cake-build/cake/blob/develop/src/Cake.Common/Tools/GitLink/GitLinkAliases.cs#L13 it is the same...